src/main/java/com/zombiecastlerush/building/Inventory.java

```diff
@@ -12,6 +12,10 @@
     public Inventory(){
     }
 
+    public void setItems(List<Item> list){
```

Comment:
- [ ] Make sure to add this to the [Game Mock-up](https://docs.google.com/presentation/d/14OPQWXwFgpWjsf6KbH2bE3556LCGIcyVj94nyy2Md6g/edit#slide=id.gdefcc1da4f_1_26)